docs/docs/env-vars.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Environment variables"
+route: "/env-vars"
+---
+
+# Environment Variables
+
+Tanka currently makes use of the following environment variables:
+* `TANKA_KUBECTL_PATH` -- path to the `kubectl` tool, defaults to `"kubectl"`.
```

Comment:
I'd prefer a more structured approach to this:

```md
#### `TANKA_KUBECTL_PATH`
**Description**: Path to the `kubectl` tool executable
**Default**: `$PATH/kubectl`
```

That way, one can also hyperlink to the section